ql/currencies/africa.hpp

```diff
@@ -161,6 +162,16 @@ namespace QuantLib {
       public:
         XOFCurrency();
     };
+
+    // Botswanan Pula
```

Comment:
May you move it up so it's in alphabetical order?  The file is easier to browse this way.  Same for the cpp.  Thanks!

Reply:
Yes, will push the sorting of all currencies shortly. Also corrected the ISO code of BWP which I had incorrectly read.

Reply:
Done!